cmd/convox/rack.go

```diff
@@ -215,7 +221,19 @@ func cmdRackParamsSet(c *cli.Context) error {
 		return stdcli.Error(err)
 	}
 
-	fmt.Println("OK")
+	if c.Bool("wait") {
+		stdcli.Startf("Waiting for completion")
+
+		// give the rack a few seconds to start updating
+		time.Sleep(5 * time.Second)
+
+		if err := waitForRackRunning(c); err != nil {
+			return stdcli.Error(err)
+		}
+
+		stdcli.OK()
```

Comment:
Think the `fmt.Println("OK")` (or something to that effect) should be incorporated for when wait isn't set.